src/DBFTPlugin/ConsensusService.cs

```diff
@@ -632,7 +635,8 @@ private void RequestChangeView(ChangeViewReason reason)
 
         private bool ReverifyAndProcessPayload(ExtensiblePayload payload)
         {
-            if (!payload.Verify(context.Snapshot)) return false;
+            var extensibleWitnessWhiteList = Blockchain.UpdateExtensibleWitnessWhiteList(system.Settings, system.StoreView);
```

Comment:
I see it was used here for verifying on recovery message.

Reply:
@bettybao1209 I think neo-project/neo#2326 can solve this problem.

Reply:
Fixed.